Library/ViewModels/LoginToutViewModel.swift

```diff
@@ -274,13 +343,20 @@ public final class LoginToutViewModel: LoginToutViewModelType, LoginToutViewMode
     self.viewIsPresentedProperty.value = isPresented
   }
 
+  fileprivate var viewWillAppearProperty = MutableProperty(())
+  public func viewWillAppear() {
+    self.viewWillAppearProperty.value = ()
+  }
+
   public let attemptFacebookLogin: Signal<(), Never>
+  public private(set) var didSignInWithApple: Signal<SignInWithAppleEnvelope, Never> = .empty
```

Comment:
Had to make this property a `var` with an empty initialization because it's only used inside the `if #available` and it's unavailable on iOS 12, and Xcode doesn't allow stored properties to be "potentially  unavailable".